### ISSUE TITLE
Help docs for --matching are unclear

### ISSUE THREAD

**Issue by oalders** [houseabsolute/ubi#39]
>   -m, --matching <matching>  A string that will be matched against the release filename when there
                             are multiple files for your OS/arch, i.e. "gnu" or "musl". Note that
                             this will be ignored if there is only used when there is only one
                             matching release filename for your OS/arch.

It's the 

>  if there is only used when there is only one matching release filename for your OS/arch

that is hard to parse.

**Comment by autarch** [comment]
It's hard to parse because it's gibberish. I will clean that up.

**Comment by autarch** [comment]
I just pushed a fix. This will be in the next release. Thanks for the report!

**Comment by oalders** [comment]
That's much clearer. Thanks! 🚀 

Closed via d7848af128425e
